packages/modal/src/Modal.js

```diff
@@ -110,22 +121,25 @@ Modal.defaultProps = {
   parentNode: document.body,
   shouldCloseOnOverlayClick: true,
   shouldCloseOnEsc: true,
-  showCloseIcon: true,
+  showCloseButton: true,
   verticalAlign: "flex-start",
   zIndex: 1500,
+  showHeader: true,
 };
 Modal.propTypes = {
   isOpen: PropTypes.bool,
   parentNode: PropTypes.instanceOf(HTMLElement),
   shouldCloseOnOverlayClick: PropTypes.bool,
   shouldCloseOnEsc: PropTypes.bool,
-  showCloseIcon: PropTypes.bool,
+  showCloseButton: PropTypes.bool,
```

Comment:
This is a breaking change and the commit should be flagged as so. This will make the next build increase the major version.

https://www.conventionalcommits.org/en/v1.0.0-beta.4/#commit-message-with-description-and-breaking-change-in-body

Reply:
closeProps to closeButtonProps as well